src/EditorFeatures/Test2/IntelliSense/VisualBasicSignatureHelpCommandHandlerTests.vb

```diff
@@ -213,7 +220,8 @@ End Class
             End Using
         End Function
 
-        <MemberData(NameOf(AllCompletionImplementations))> <WpfTheory, Trait(Traits.Feature, Traits.Features.SignatureHelp)>
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.SignatureHelp)>
```

Comment:
wrapping all these just made reviewing this more difficult.  i would ask that for these complex reviews we minimize unnnecessary changes like this.

Reply:
OK. Let us revert.

---
In reply to: [232508362](https://github.com/dotnet/roslyn/pull/29016#discussion_r232508362) [](ancestors = 232508362)